tools/static-assets/skel-full/imports/ui/components/info/info.js

```diff
@@ -1,8 +1,10 @@
+import { Links } from '/imports/api/links/links.js';
+import { Meteor } from 'meteor/meteor';
 import './info.html';
-import { Links } from '/imports/api/links/links.js'
-import { Meteor } from 'meteor/meteor'
 
-Template.info.onCreated(function infoOnCreated() {
+/* eslint-disable func-names, prefer-arrow-callback */
```

Comment:
I find it slightly weird to have this (or any of these) hovering mid-file.  Perhaps hoist all of these to the top?

But on that note, maybe the `eslint` stuff should just be removed until it's figured out what the best way to do it is?  With having these `eslint-disable` lines here (And thus implying an `eslint` "setup" is in place), it almost seems like setting up the new `package.json` with an `eslintConfig` and incorporate `eslint-plugin-meteor`, etc. might be better.

Anyhow, everything else looks great.